src/app/plugins/register.plugin.ts

```diff
@@ -24,6 +25,19 @@ export class RegisterPlugin extends Plugin {
     const successfulClasses: string[] = [];
     const invalidClasses: string[] = [];
     for (const arg of args) {
+      if (arg === 'all') {
+        const member = this.container.guildService.get().members.get(message.author.id);
+        if (!member) {
+          continue;
+        }
+
+        const isModerator = MemberUtils.hasRole(member, 'Moderator');
```

Comment:
maybe only expose hasRole through guildService

```js
const isModerator = this.container.guildService.memberHasRole(message.author, '...');
```

... or some other way such that we don't have a repeated "member lookup" phase.